Makefile

```diff
@@ -33,7 +33,7 @@ clean:
 	rm -fr *.o build deps/*/*.o example example.dSYM src/*.o
 
 example: build
-	$(CC) $(CFLAGS) -Ibuild/include -o example example.c -Lbuild/lib -lgraph
+	$(CC) $(CFLAGS) -Ibuild/include -o example example.c -Lbuild/lib -lgraph -lm -lpthread
```

Comment:
@nderjung This is unnecessary given that `-lm` and `-lpthread` are already set in the `CFLAGS` variable. Any specific reason why you did this?

Reply:
For some reason, this didn't work for me, it looked like the order mattered for `gcc`.  While it was set in `CFLAGS`, it gave me a complaint unless I placed the libraries before the build output:

```
build/lib/libgraph.a(graph.o): In function `_graph_item_set_label_th':
graph.c:(.text+0x24c): undefined reference to `log10'
graph.c:(.text+0x251): undefined reference to `floor'
graph.c:(.text+0x27c): undefined reference to `log10'
graph.c:(.text+0x281): undefined reference to `floor'
build/lib/libgraph.a(graph.o): In function `graph_add_edge':
graph.c:(.text+0x37c): undefined reference to `pthread_create'
graph.c:(.text+0x3ab): undefined reference to `pthread_join'
build/lib/libgraph.a(graph.o): In function `graph_add_vertex':
graph.c:(.text+0x5e8): undefined reference to `pthread_create'
graph.c:(.text+0x61c): undefined reference to `pthread_join'
build/lib/libgraph.a(graph.o): In function `graph_new':
graph.c:(.text+0xc02): undefined reference to `pthread_create'
graph.c:(.text+0xc72): undefined reference to `pthread_join'
collect2: error: ld returned 1 exit status
Makefile:36: recipe for target 'example' failed
make: *** [example] Error 1
```